app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -32,52 +76,182 @@ export default {
+        {
+          id: 'default',
+          title: this.$t('INBOX_MGMT.WIDGET_BUILDER.WIDGET_SCREEN.DEFAULT'),
+          checked: true,
+        },
+        {
+          id: 'chat',
+          title: this.$t('INBOX_MGMT.WIDGET_BUILDER.WIDGET_SCREEN.CHAT'),
+          checked: false,
+        },
+      ],
+      isDefaultScreen: true,
+      isWidgetVisible: true,
+    };
+  },
   computed: {
     getWidgetHeadConfig() {
       return {
         welcomeHeading: this.welcomeHeading,
-        welcomeTagLine: this.welcomeTagLine,
+        welcomeTagline: this.welcomeTagline,
         websiteName: this.websiteName,
-        websiteDomain: this.websiteDomain,
         logo: this.logo,
-        isExpanded: this.isExpanded,
+        isDefaultScreen: this.isDefaultScreen,
         isOnline: this.isOnline,
-        replyTime: this.replyTime,
+        replyTime: this.replyTimeText,
+        color: this.color,
       };
     },
+    getWidgetBodyConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        isOnline: this.isOnline,
+        replyTime: this.replyTimeText,
+        color: this.color,
+      };
+    },
+    getWidgetFooterConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        color: this.color,
+      };
+    },
+    replyTimeText() {
+      switch (this.replyTime) {
+        case 'in_a_few_minutes':
+          return this.$t(
+            'INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_MINUTES'
+          );
+        case 'in_a_few_hours':
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_HOURS');
+        case 'in_a_day':
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_DAY');
+        default:
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_HOURS');
+      }
+    },
+  },
+  methods: {
+    handleScreenChange(item) {
+      this.isDefaultScreen = item.id === 'default';
+    },
+    toggleWidget() {
+      this.isWidgetVisible = !this.isWidgetVisible;
+      this.isDefaultScreen = true;
+    },
   },
 };
 </script>
 
 <style lang="scss" scoped>
-.text-lg {
-  font-size: var(--font-size-default);
+.screen-selector {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 .widget-wrapper {
-  box-shadow: var(--shadow-larger);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0 0px 20px 5px rgb(0 0 0 / 10%);
   border-radius: var(--border-radius-large);
-  background-color: var(--color-background);
-  z-index: 99;
+  background-color: #f4f6fb;
+  min-width: 32rem;
+  max-width: 32rem;
+
+  .branding {
+    padding-top: var(--space-one);
+    padding-bottom: var(--space-one);
+    text-align: center;
+  }
+}
+.widget-bubble {
+  display: flex;
+  flex-direction: row;
+  margin-top: var(--space-normal);
+  min-width: 32rem;
+  max-width: 32rem;
+
+  .bubble {
+    display: flex;
+    align-items: center;
+    border-radius: 3rem;
+    height: 6rem;
```

Comment:
Use css variables here - if not present we can just multiply them - `(--space-three * 2)`

Reply:
Done